GenderPayGap.WebJob/Functions/Functions.SendReminderEmails.cs

```diff
@@ -65,7 +65,6 @@ public void SendReminderEmails([TimerTrigger("0 * * * *")] TimerInfo timer)
             catch (Exception ex)
             {
                 CustomLogger.Error("Failed to send reminder emails", ex);
-                throw;
```

Comment:
I think this was throwing the exception that crashed the webjob. The intended behaviour is that if one fails, it should log the problem but then move on to the next user. But I don't see this error message in the Azure logs...

Reply:
But we do see the error on line 154,  "Failed to send reminder email to Notify.". That exception does get thrown, so I'd expect to end up in the catch block on line 95. But we don't see the "Failed whilst sending or saving reminder email." error message

Reply:
If you want to catch an error for each user individually, the try-catch will need to be within the for-each loop